ocaml/idl/datamodel_roles.ml

```diff
@@ -6,9 +6,12 @@ let role_vm_power_admin = "vm-power-admin"
 let role_vm_admin = "vm-admin"
 let role_vm_operator = "vm-operator"
 let role_read_only = "read-only"
+let role_client_cert = "client-cert"
```

Comment:
I think these are roughly sorted by level of privilege, so this should go near pool operator here.

Reply:
I wanted to keep the original set of roles together.